mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -22,12 +22,31 @@ odoo.define('mail_tracking.partner_tracking', function(require){
         hasPartnerTrackings: function () {
             return false;
         },
+
+        /**
+         * Messages do not have any email Cc.
+         *
+         * @return {boolean}
+         */
+        hasEmailCc: function () {
+            return false;
+        },
+
+        /**
+         * Email are not Cc.
+         *
+         * @return {boolean}
+         */
+        isEmailCc: function (email) {
+            return false;
+        },
     });
 
     Message.include({
```

Comment:
I wonder why can't all this code go into `AbstractMessage`? It seems to include sane defaults that will `return false` when a message has no CC :thinking:

Reply:
To ensure that new implementations doesn't break the xml. I don't see the 'real world' scenario... but perhaps other like reuse AbstractMessage with the same xml.